presto-docs/src/main/sphinx/installation/deploy-docker.rst

```diff
@@ -48,81 +46,56 @@ verify your Docker setup by running the command in step 4.
 
-3. To start the Presto CLI (command-line interface), open a new terminal window and run the following command:
-
-   ``docker exec -it presto presto-cli``
-
-   The presto prompt is shown:
-
-   ``presto>``
-
-   If the ``docker exec`` command returns an error similar to the following:
-   
-   ::
-    
-    "docker exec" requires at least 2 arguments.
-    See 'docker exec --help'.
-    Usage:  docker exec [OPTIONS] CONTAINER COMMAND [ARG...]
-    Execute a command in a running container``
+   Downloading the container may take a few minutes. When the download completes, go on to the next step.
 
-   the Presto server has not completed starting. Wait a little and try again, until the ``presto>`` prompt is shown.
+2. On your local system, create a file named ``config.properties`` containing the following text: 
 
-4. To verify that your Presto server and Presto CLI are communicating together, use the Presto CLI to query your local Presto server. Run the following command at the ``presto>`` prompt:
+   .. code-block:: none
 
-   ``show catalogs;``
+    coordinator=true
+    node-scheduler.include-coordinator=true
+    http-server.http.port=8080
+    discovery-server.enabled=true
+    discovery.uri=http://localhost:8080
 
-   The response should be similar to the following:
+3. On your local system, create a file named ``jvm.config`` containing the following text: 
 
-   ::
+   .. code-block:: none
 
-    presto> show catalogs;
-    Catalog
-    ---------
-    jmx
-    memory
-    system
-    tpcds
-    tpch
-    (5 rows)
-    
-    Query 20230614_181140_00000_uutw6, FINISHED, 1 node
-    Splits: 19 total, 19 done (100.00%)
-    [Latency: client-side: 0:13, server-side: 0:11] [0 rows, 0B] [0 rows/s, 0B/s]
-    
-    presto>
+    -server
+    -Xmx2G
+    -XX:+UseG1GC
+    -XX:G1HeapRegionSize=32M
+    -XX:+UseGCOverheadLimit
+    -XX:+ExplicitGCInvokesConcurrent
+    -XX:+HeapDumpOnOutOfMemoryError
+    -XX:+ExitOnOutOfMemoryError
+    -Djdk.attach.allowAttachSelf=true
+     
+4. To start the Presto server in the Docker container, run a command similar to the following example:
 
-5. To use the Presto CLI in your local Docker image to connect to a remote Presto server, run the following command: 
+   ``docker run -p 8080:8080 -it -v </local/path/to/config.properties>:/opt/presto-server/etc/config.properties -v </local/path/to/jvm.config>:/opt/presto-server/etc/jvm.config --name presto prestodb/presto:0.284``
 
-   ``docker exec -it presto presto-cli --server Presto-endpoint-URL --user username --password``
+   Modify this example and replace ``</local/path/to/config.properties>`` and ``</local/path/to/jvm.config>`` with the paths to the ``config.properties`` and ``jvm.config`` files that you created.
 
-   You will be prompted to enter the password for the Presto user, then the ``presto>`` prompt is shown.
+   This command also assigns the name ``presto`` for the downloaded image ``prestodb/presto:0.284``.
```

Comment:
```suggestion
   This command also assigns the name ``presto`` for the newly-created container that uses the downloaded image ``prestodb/presto:0.284``.
```

I guess this would be more accurate :-)